assets/src/theme/p4ServerThemes.js

```diff
@@ -1,4 +1,4 @@
-/* global wp */
+// global wp
```

Comment:
I'm not sure we actually need to declare this globally here 🤔 We use `wp` in lots of places without it!